.azure-pipelines.yml

```diff
@@ -49,7 +49,7 @@ stages:
     variables:
       install: ''
       configure: ''
-      tests: ''
+      tests: '!433'
```

Comment:
will be overridden by the torture test, I would rather add it here:
https://github.com/curl/curl/pull/6777/files#diff-ba8565e0a98087480c3c079dff9a54fc96be8f7d3ffed7aeb4c6ff60bd135ce9R92

Reply:
Yes, but we haven't seen a problem in the torture tests so I figured it would be fine... Torture tests don't need things to evaluate correctly the same way, they just need to not crash and not leak.